components/sprayproxy/production/kustomization.yaml

```
@@ -2,11 +2,11 @@ apiVersion: kustomize.config.k8s.io/v1beta1
 kind: Kustomization
 resources:
   - ../base
-  - https://github.com/redhat-appstudio/sprayproxy/config?ref=c6f8bf26007c140d51326760e16eb52ae5072e3b
+  - https://github.com/konflux-ci/sprayproxy/config?ref=c6f8bf26007c140d51326760e16eb52ae5072e3b
   - pipelines-as-code-secret.yaml
 
 images:
-  - name: ko://github.com/redhat-appstudio/sprayproxy
+  - name: ko://github.com/konflux-ci/sprayproxy
```

The manifest in https://github.com/konflux-ci/sprayproxy/config at commit c6f8bf26007c140d51326760e16eb52ae5072e3b are still using `ko://github.com/redhat-appstudio/sprayproxy` as image name so kustomize is no longer replacing the name and tag.

This line must be reverted until manifest in sprayproxy use new org name
```suggestion
  - name: ko://github.com/redhat-appstudio/sprayproxy
```

---

Good catch. Thank you.